packages/cli/register/src/ir-to-fdr-converter/convertPackage.ts

```diff
@@ -600,9 +608,36 @@ function convertHttpEndpointExample(
                 throw new Error("Unknown ExampleResponse: " + irExample.response.type);
             }
         }),
-        responseBody: irExample.response.body?.jsonExample,
-        responseBodyV3:
-            irExample.response.body != null ? { type: "json", value: irExample.response.body.jsonExample } : undefined,
+        responseBody: irExample.response._visit({
```

Comment:
responseBody is deprecated anyway in favor of responseBodyV3, so this part of the code is technically useless (and should be deletd)